src/main/java/com/twilio/sdk/resource/instance/taskrouter/WorkspaceStatistics.java

```diff
@@ -242,6 +245,35 @@ public Integer getTotalWorkers() {
 	}
 
 	/**
+	 * Get the number of tasks for each priority
+	 *
+	 * @return the number of tasks with priority p
+	 */
+	public Integer getTasksWithPriority(int priority) {
```

Comment:
how come we need to add the same code 3 different places? is it possible to simplify this?